Source/AppScaffolding/AppScaffolding.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
-    <Version>11.4.1.1</Version>
+    <Version>11.5</Version>
```

Comment:
For stupid legacy reasons that I swear I'll fix one day (don't ask how long I've been telling myself this), please always use 4 numbers. Eg: 11.5.0.1. Don't worry if you don't get around to this; I'll fix it if you don't.